pyuvdata/uvbeam/uvbeam.py

```diff
@@ -607,6 +633,128 @@ def __init__(self):
 
         super(UVBeam, self).__init__()
 
+    @property
+    def _freq_params(self):
+        """List of strings giving the parameters that shaped like the freq_array."""
+        return [
+            "bandpass_array",
+            "receiver_temperature_array",
+            "loss_array",
+            "mismatch_array",
+            "freq_array",
+        ]
```

Comment:
yeah, that's a good suggestion.

Reply:
Turned out to be a little more complicated than I expected. It can't be a property because of too much recursion (since I am using the `__iter__` method to get the UVParameters) and you have to be careful to grab the freq_array form before you change it.